cloud_info/providers/openstack.py

```diff
@@ -355,6 +397,56 @@ def get_images(self):
             images[img_id] = aux_img
         return images
 
+    @_rescope
+    def get_instances(self, os_project_name=None, **kwargs):
```

Comment:
😎Are we really publishing this information? I guess it can be valuable for security teams to know if a given image is running

Reply:
We don't publish detailed information but we publish numbers: https://github.com/EGI-Foundation/cloud-info-provider/blob/2d0fe14c8fecf91062d8fd4a9678987912de07bf/etc/templates/compute.glue21#L54-L57
as computed by https://github.com/EGI-Foundation/cloud-info-provider/blob/2d0fe14c8fecf91062d8fd4a9678987912de07bf/etc/templates/compute.glue21#L21-L24

Reply:
In the GLUE 2.1 cloud model there is such a relation:
![glue-2 1-cloud-computing-model](https://user-images.githubusercontent.com/451417/44841442-a40e7f80-ac43-11e8-888f-d9540da8e4f7.png)

Reply:
I was looking at the table with the description and if I'm not wrong it's missing there. I have added a comment in the spec draft